### ISSUE TITLE
Incorrect labelling of "sort code"

### ISSUE THREAD

**Issue by mbrookes** [woocommerce/woocommerce#28844]
**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
The store country in admin and customer’s billing country in the front-end are used to determine the term for ‘Sort Code’.

This was introduced in https://github.com/woocommerce/woocommerce/pull/6814

Unfortunately this doesn't quite solve the problem, since the label should be specific to where the bank account is located, not where the customer or store are located.

A US account has a routing number (ACH or wire). It isn't called a sort code, even if the customer is in the UK, or anything else anywhere else. Calling it such is confusing at best, and at worst could prevent the customer from completing payment. ("Sort code" for example is specific the the UK banking system – as is BACS (Bacs?) for that matter.)

**Expected behavior**
The label should be specific to where the bank account is located, possibly through an additional field in the account details list.

**Actual behavior**
A clear and concise description of what actually happens. Please be as descriptive as possible; 

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Configure direct bank transfer
2. Create an order
3. Check out
4. Note that the routing number is labelled according the billing country, so may by incorrectly labelled, depending on the bank account country and payment system used.

**Screenshots**
UK Billing Country, US Bank Account:
![Screenshot 2021-01-19 at 00 57 25](https://user-images.githubusercontent.com/357702/104975138-c1e60600-59f1-11eb-8256-c212654c552a.png)

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://themes-backend.material-ui.com
Site address (URL): https://themes-backend.material-ui.com
WC Version: 4.8.0
REST API Version: ✔ 4.8.0
WC Blocks Version: ✔ 3.8.1
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.7.3
Log Directory Writable: ✔
WP Version: 5.6
WP Multisite: –
WP Memory Limit: 768 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –
```
</details>

**Comment by juliaamosova** [comment]
Hi @mbrookes,

Thank you for taking the time to share this idea, we really appreciate your help. 

While this is clearly a great improvement/idea we won’t be able to tackle it in the upcoming weeks. We’re going to add it to our backlog where it will be considered for future releases.

**Comment by vbelolapotkov** [comment]
@pierorocca, this is payment UI improvement request. Sending to you for further prioritization.

**Comment by mbrookes** [comment]
It would be quite generous to call this an improvement request. https://github.com/woocommerce/woocommerce/pull/6814 is basically a regression, and a nine year old one at that! It should never have been merged.

**Comment by pierorocca** [comment]
I can replicate this @vbelolapotkov. However this doesn't sit within the WooPayments gateway as it's an offline bank transfer method similar to the core Check and Cash payment options. This needs to be assigned to the core team and is a simple fix. The terminology must be based on the merchant's locale.

**Comment by mbrookes** [comment]
No, it shouldn't be based on the merchant's locale. A merchant in one country can bank and have payment methods in another. In our case we're HQ'd in France, but take USD payments to a U.S. account (Wise). Shocking that this hasn't even been correctly triaged never mind fixed after so many years.

**Comment by pierorocca** [comment]
A Wise account is a multi-currency non-bank account. Wise sits between users/businesses and the banking system in order to facilitate their business model to move money without actually moving money across borders. It's a clever way to create the illusion you have bank accounts in other currencies.

> 12.2 Wise is not a bank, and your Wise Account is not a bank account. The funds held with Wise, including balances in your Wise Account, are not insured by any deposit protection scheme, including the Financial Services Compensation Scheme (FSCS). Wise holds your funds in permissible investments in accordance with applicable laws. Wise owns the interest or other earnings on these investments, if any.
>
> 12.3 Account Details. When Wise provides you with Account Details (as described further below), these Account Details are a means to receive funds from third parties into a bank account held by Wise and its affiliates. Wise can credit your Wise Account when you use these Account Details, and they are not a bank account number for a bank account held by you


Good point. I'll clarify the default should be based on the merchant's banking domicile which for real bank accounts is almost always the store country. Since there's no way to detect this for an offline method, and because of Wise's unique model a Country option could be introduced on the admin UI side. 

Note for shoppers, the terminology is based on the shopper's billing address. Do you have an issue with the shopper terminology? It sounds like you deal a lot with Wise who through their unique model allow cross-border payments that can't be done through typical bank transfers which are domestic/intra EEA only.

> The terminology must be based on the merchant's locale.

**Comment by pierorocca** [comment]
i.e. what you're really after here is a Wise specific solve, not a solve for traditional domestic bank transfers.

**Comment by pierorocca** [comment]
If that's the case, searching for Wise turns up this [well rated Wise gateway](https://co.wordpress.org/plugins/wc-wise-gateway/) from wordpress.org
